recipes/examples/fireworks_functions_qa.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
could you move it to `recipes/examples/function_calling/fireworks_functions_qa.ipynb`?